ES/ESdd021.xml

```diff
@@ -314,9 +179,321 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        <title type="complete" ref="LIT3509Epistle"/>
+                        <note>Correct folio order for 1 John: 1:1–2:23 (<locus from="9ra" to="10vb"/>); 2:24–4:17
+                           (<locus from="14ra" to="16vb"/>); 4:17–5:20 (<locus from="21ra" to="22vb"/>).</note>
+                     </msItem>
+                     
+                     <msItem xml:id="p3_i1.4">
+                        <locus from="22vb"/>
+                        <locus from="25ra" to="25va"/>
+                        <title type="complete" ref="LIT3510Epistle"/>
+                        <note>Correct folio order: 2 John: 1:1–4 (<locus target="#22vb"/>); 1:4–13 (<locus from="25ra" to="25va"/>).</note>
+                     </msItem>
+                     
+                     <msItem xml:id="p3_i1.5">
+                        <locus from="25va" to="26va"/>
+                        <title type="complete" ref="LIT3511Epistle"/>
+                     </msItem>
+                     
+                     <msItem xml:id="p3_i1.6">
+                        <locus from="26va" to="33ra"/>
+                        <title type="complete" ref="LIT3512Epistle"/>
+                     </msItem>
+                     
+                     <msItem xml:id="p3_i1.7">
+                        <locus from="33ra" to="35ra"/>
+                        <title type="complete" ref="LIT3513Epistle"/>
+                     </msItem>
+                  </msItem>
+                  
+                  <msItem xml:id="p3_i2">
+                     <locus from="35va" to="93vb"/>
+                     <title type="complete" ref="LIT1019Actsof"/>
+                     <note>The text is divided into 63  ‘sections’ (<foreign xml:lang="gez">kǝfl</foreign>) .</note>
+                  </msItem>
+               </msContents>
+               
+                  <physDesc>
+                     <objectDesc form="Codex">
+                        
+                        <supportDesc>
+                           <support>
+                              <material key="parchment"/>
+                           </support>
+                           <extent><measure unit="leaf" quantity="86">86</measure>
+                              <locus from="6r" to="94v"/>
+                           </extent>
+                           <condition>The manuscript shows signs of wear, particularly on the front folia. Some folia are out of order. In some areas, the ink has faded. 
+                              There is a hole in <locus target="#20"/>. The folia are stained with water and wax.
+                           </condition>
+                        </supportDesc>
+                        
+                     <layoutDesc>
+                        <layout columns="2" writtenLines="20">
+                           <note>The number of lines differs: from 20 (<locus target="#11r"/>) to 23 (<locus target="#63r"/>).</note>
+                           <dimensions unit="mm">
```

Comment:
Does the first set of dimensions in each layout refer to the text area? (here and in the other msParts)  If so, could you provide a statement and xml:id such as the following from the guidelines
```
   <note corresp="#textarea2 #margin2">Data on text area and margin dimensions taken from <locus target="#119r"></locus>.</note>
                   <dimensions unit="mm" xml:id="textarea2">
                     <height>145</height>
                     <width>100</width>
                   </dimensions>
```